test/libsolidity/syntaxTests/nameAndTypeResolution/290_varM_disqualified_as_keyword.sol

```diff
@@ -7,5 +7,3 @@ contract test {
 }
 // ----
 // DeclarationError: (50-55): Identifier not found or not unique.
```

Comment:
Sorry, I think since this isn't reporting 3 errors now, maybe we should split the test case to check for all 3? Is it worth it?

Reply:
True, makes sense

Reply:
I'd keep 3 copies of 290 to keep them grouped